backend/src/payments/services/stripe.service.ts

```diff
@@ -38,8 +47,12 @@ export class StripeService {
           },
         ],
         mode: 'payment',
-        success_url: this.configService.get('STRIPE_SUCCESS_URL'),
-        cancel_url: this.configService.get('STRIPE_CANCEL_URL'),
+        success_url: `${this.configService.get(
+          'STRIPE_SUCCESS_URL',
```

Comment:
This means the STRIPE_CANCEL_ENV variable can be removed, no?

Reply:
It is already dupliacted with the "FRONTEND_DOMAIN" env var...we dont need 3 Vars containing almost the exact same value